src/Umbraco.Core/Services/NotificationService.cs

```diff
@@ -204,6 +204,8 @@ public Notification CreateNotification(IUser user, IEntity entity, string action
             if (createSubject == null) throw new ArgumentNullException("createSubject");
             if (createBody == null) throw new ArgumentNullException("createBody");
 
+            allVersions = allVersions.OrderBy(x => x.UpdateDate).ToArray();
```

Comment:
@clausjensen please review this update: 4fa1a3948e24eaa4750823975baf6c59060de3e0